controlplane/test/test-util.ts

```diff
@@ -227,24 +240,52 @@ export const SetupKeycloak = async ({
   realmName: string;
 }) => {
   await keycloakClient.authenticateClient();
-  await keycloakClient.client.realms.create({
-    realm: realmName,
-    enabled: true,
-    displayName: realmName,
-    registrationEmailAsUsername: true,
-  });
-  const id = await keycloakClient.addKeycloakUser({
-    email: userTestData.email,
-    realm: realmName,
-    isPasswordTemp: false,
-    password: 'wunder@123',
-    id: userTestData.userId,
-  });
-  await keycloakClient.seedGroup({
-    realm: realmName,
-    userID: id,
-    organizationSlug: userTestData.organizationSlug,
-  });
+
+  try {
+    await keycloakClient.client.realms.create({
+      realm: realmName,
+      enabled: true,
+      displayName: realmName,
+      registrationEmailAsUsername: true,
+    });
+  } catch (e: any) {
+    if (e.response?.status !== 409) {
+      throw e;
```

Comment:
Please attach more informations to the errors to make debugging easier.

Reply:
Done